app/routes/my_account.js

```diff
@@ -1,2 +1,22 @@
-import dashboard from "./my_dashboard";
-export default dashboard.extend({});
+import accountDetails from "./account_details";
+
+export default accountDetails.extend({
+  setupController(controller, model, transition) {
+    this._super(...arguments);
+    this.controllerFor("application").set("showSidebar", true);
+    this.controllerFor("application").set(
```

Comment:
I agree entirely. Although there's no clear way of doing this at the moment.
Would there be a way for the controller to have a `title` property, which gets read and populates the title ? In the case where the controller doesn't have a title it would default to the one from the application controller.

pseudo code 
```
<template>
   <div class="title">{{ currentPage.title or baseTitle  }}</div>
</template>
```

Reply:
Yeah, right that's why i did it the way its done. Currently, there's only this way, for us to implement something like Patrick's snippet in above example, this would require a whole app's change.

Reply:
I don't think @patrixr 's suggestion requires change is whole app. It just checks if controller of current page doesn't have title then user Application controller's title

Reply:
Sure then, I'll try this. and make a change.

Reply:
Yeah, I think we just need to add a small piece of code in the root controller that does that

Reply:
wouldn't it be better to create a component for title bar and depending on the current page add condition to change the text.

Reply:
That would a solution if each page includes a title element.

The way I believe it's set up is like this :
```
<app>
    <title> {{ title }} </title> . <!-- This element is always present above the current page -->
   <current_page> 
       <!-- no title within the page -->
       <content></content>
   </current_page>
</app>
```